.gitignore

```diff
@@ -6,3 +6,6 @@
 /esdoc
 /node_modules
 /demo/kinto*
+/.vscode
+.DS_Store
```

Comment:
I think these should be added to users global `.gitconfig` instead (eg. `~/.gitconfig`). We're not gonna ignore all possible kinds of stuff like this :)